pkg/server/batch_check.go

```diff
@@ -87,32 +87,33 @@ func (s *Server) BatchCheck(ctx context.Context, req *openfgav1.BatchCheckReques
 		methodName,
 	).Observe(queryCount)
 
+	var batchResult = map[string]*openfgav1.BatchCheckSingleResult{}
+	for correlationID, outcome := range result {
+		batchResult[string(correlationID)] = transformCheckResultToProto(outcome)
+		s.emitCheckDurationMetric(outcome.CheckResponse.GetResolutionMetadata(), "batch_check")
```

Comment:
nit: maybe for consistency this should use `methodName` from
above

Reply:
going to merge this PR as is but will work this feedback into [the other PR](https://github.com/openfga/openfga/pull/2102#discussion_r1870373306) you just reviewed since it needs updated error handling anyway